.github/CODEOWNERS

```
@@ -1,2 +1,2 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
-.github/CODEOWNERS @sonarsource/sonarcloud
+.github/CODEOWNERS @Sonarsource/analysis-experience-squad
```

The org should be lowercase
```suggestion
.github/CODEOWNERS @sonarsource/analysis-experience-squad
```

---

Looks like both do not work. Will get in touch with Dev-Infra/ITOPs

---

No worries - DevInfra needs to add some explicit permissions to solve the warning. I sent a request already for this repo https://sonarsource.atlassian.net/browse/BUILD-4707